src/components/AddPlaidBankAccount.js

```diff
@@ -123,9 +124,9 @@ class AddPlaidBankAccount extends React.Component {
                             <ActivityIndicator color={themeColors.spinner} size="large" />
                         </View>
                     )}
-                    {Boolean(this.props.plaidData.error) && (
+                    {(Boolean(this.props.plaidData.error) || Boolean(this.props.plaidData.errors)) && (
```

Comment:
We could simplify this by adding a const variable at the top of the render method which you could call something like `plaidDataErrorMessage`. Set it to the message from `.error` or the first value in `.errors` using lodash get. Then we can conditionally render based off of it and show `plaidDataErrorMessage` inside the text component.